server.js

```diff
@@ -24,10 +16,58 @@ app.use(bodyParser.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Endpoints = /shows & /titles/title & year/year & cast/cast & genre/listed_in & country/country&type=Movie or TV Show')
+})
+
+app.get('/shows', (req, res) => {
+  res.json(data)
+})
+
+app.get('/titles/:title', (req, res) => {
```

Comment:
the good REST api provides both: 

1. the list of something on path: `something/`
2. the single elements details on path: `something/:title`, `something/:id etc`

Here you have list of shows in `/shows` and the details under `/titles/:title`, you should make it `shows/:title`  or add separate endpoint `/titles` to list for example only titles from the data set.